docs/usage/node.md

```diff
@@ -8,6 +8,10 @@ description: Node.js versions support in Renovate
 Renovate can upgrade the [Node.js](https://nodejs.org/en/) runtime used by your project.
 This way you're using the latest bug fixes, performance improvements, security mitigations, etc.
 
+## LTS codenames
+
+Renovate understands [codenames for Node.js LTS releases](https://github.com/nodejs/Release/blob/main/CODENAMES.md) and will offer upgrades for them (e.g. from `fermium` to `gallium`) as long as `node` versioning scheme is being used.
```

Comment:
```suggestion
Renovate understands [codenames for Node.js LTS releases](https://github.com/nodejs/Release/blob/main/CODENAMES.md) and will offer upgrades for them (e.g. from `fermium` to `gallium`) as long as the `node` versioning scheme is being used.
```

Reply:
@viceice I applied this suggestion right after you triggered auto-merge 😬